helm/default-apps-cloud-director/values.yaml

```diff
@@ -1,6 +1,6 @@
-clusterName: ""
-organization: ""
-managementCluster: ""
+clusterName: "dummy"
```

Comment:
The default values must be empty.

Reply:
These are required by the new schema [validation](https://github.com/giantswarm/default-apps-cloud-director/pull/60/files#diff-c70a68e380af0b06885a3e05a8062ded335ec477479b13ebecfdc9f016d7b97dR34) to succeed. These values will be overriden by the app operator.

Reply:
@marians What should be our approach when there is no default value for a parameter and it is required for the chart?